### ISSUE TITLE
Make search page actions more fun with animations

### ISSUE THREAD

**Issue by JmillsExpensify** [Expensify/App#52372]
We've on the cusp of adding animations for pay and approve via [this issue](https://github.com/Expensify/App/issues/49847), which is super cool. We're also very close to merging a PR for approve/pay report actions on the `Search` page. Here's how `Search` approvals look today (this is the bulk case, you can also approve or pay a report individually). 

![CleanShot 2024-11-12 at 11 33 41](https://github.com/user-attachments/assets/8fc2e4f7-dd74-4a4f-ae0a-acdb25629ef4)

@Expensify/design Shall we use the exact same animations from #49847 on the `Search` page as well? Let's use this issue to hash things out.

**Comment by JmillsExpensify** [comment]
Not the highest priority but still on the list when the design team has a free moment

**Comment by shawnborton** [comment]
Will try to take a look at this one this week, sorry for the delay!

**Comment by shawnborton** [comment]
Sorry, no progress yet.

**Comment by shawnborton** [comment]
No update

**Comment by JmillsExpensify** [comment]
No problem

**Comment by shawnborton** [comment]
Oof still dragging my feet here, sorry

**Comment by JmillsExpensify** [comment]
Sounds good to me.

**Comment by shawnborton** [comment]
Still not prioritizing this one yet.

**Comment by shawnborton** [comment]
Same, not a priority.

**Comment by JmillsExpensify** [comment]
Looks like this is still in the queue

**Comment by JmillsExpensify** [comment]
Moved to monthly since this isn't a priority.

**Comment by shawnborton** [comment]
Sorry, no update still. Let me know when you think we should pick it back up.

**Comment by shawnborton** [comment]
No update, I think we can pick this back up once we get through with Suggested Searches.

**Comment by JmillsExpensify** [comment]
Thoughts on thinking about this level of polish now?

**Comment by JmillsExpensify** [comment]
Yeah, that's a fair point for the moment. They aren't fully offline compatible. I think that will change though once we refactor the Search page. Also fine if you want to close it btw. I don't think this is the highest thing out there.

**Comment by dannymcclain** [comment]
>I say we close and focus on other items?

Down with this!

**Comment by shawnborton** [comment]
Word, gonna do it 👍
